ci/docker/Dockerfile

```diff
@@ -22,14 +22,18 @@ ENV ANDROID_HOME /opt/android-sdk-linux
 ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
 ENV ANDROID_SDK_INSTALL_COMPONENT "echo \"y\" | \"$ANDROID_HOME\"/tools/bin/sdkmanager --verbose"
 
+# Give all users access to Android SDK (otherwise build_user won't be able to access it).
 RUN mkdir -p $ANDROID_HOME && \
     curl https://dl.google.com/android/repository/$ANDROID_SDK_FILE_NAME --progress-bar --location --output $ANDROID_SDK_FILE_NAME && \
-    unzip $ANDROID_SDK_FILE_NAME -d $ANDROID_HOME && \
+    echo "Unzipping Android SDK" && \
+    unzip -qq $ANDROID_SDK_FILE_NAME -d $ANDROID_HOME && \
     rm $ANDROID_SDK_FILE_NAME && \
-    eval $ANDROID_SDK_INSTALL_COMPONENT '"tools"' && \
-    eval $ANDROID_SDK_INSTALL_COMPONENT '"platform-tools"' && \
-    eval $ANDROID_SDK_INSTALL_COMPONENT '"build-tools;25.0.2"' && \
-    eval $ANDROID_SDK_INSTALL_COMPONENT '"platforms;android-25"'
+    echo "Installing Android SDK components" && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"tools"' > /dev/null && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"platform-tools"' > /dev/null && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"build-tools;25.0.2"' > /dev/null && \
+    eval $ANDROID_SDK_INSTALL_COMPONENT '"platforms;android-25"' > /dev/null && \
+    chmod -R 777 "$ANDROID_HOME"
```

Comment:
Moved it here, previously we gave ownership of this folder to the build user in entrypoint script which was slowing down iterative testing as it was running every time on container startup

Reply:
`chmod a+rwx`, please.

Reply:
777, ruly?

@nostra13 sing it to @ming13:

Chmod777:

>Podnyal chmoda, stali drugimi prava
Stali schitatsya so mnoy
Znayut kto teper chown

>Podnyal chmoda, filesystem v liniu dala
U tebya talant bratan
Kkakoy?
Nastraivat linuxa

https://www.youtube.com/watch?v=qP303vxTLS8